omod-dstu3/src/main/java/org/openmrs/module/fhir2/providers/AllergyIntoleranceDstu3FhirResourceProvider.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.providers;
```

Comment:
Thinking about this some more, I think that instead of naming every `ResourceProvider` `...DStu3ResourceProvider` we should move them into a `r3` package so, for instance, this class would become `org.openmrs.module.fhir2.providers.r3.AllergyIntoleranceFhirResourceProvider`.